### DISCUSSION TITLE
Missing config.h in motion.h

### DISCUSSION THREAD

**Discussion by victoryred** [Motion-Project/motion#1553] [Q&A] [answered]
I've searched my motion/src directory and every place else but I can't find the config.h header referenced in motion.h. Can someone *me in the right direction?

**Comment by Mr-Dave** [comment, marked as answer]
It is generated as part of the ```autoreconf``` and ```configure``` steps of the builds.  

Run those two steps in the [build from source](https://motion-project.github.io/motion_build.html#Configure_Script)  and the file will be created.

**Reply by victoryred** [reply]
Thanks Mr-Dave! I appreciate your help.
